changelogs/CHANGELOG-1.13.md

```diff
@@ -64,6 +64,9 @@ To fix CVEs and keep pace with Golang, Velero made changes as follows:
 ### Limitations/Known issues
 * The backup's VolumeInfo metadata doesn't have the information updated in the async operations. This function could be supported in v1.14 release. 
 
+### Note
+* Velero may use more memory due to the introduction of informer-cache, increase the memory limit if restoring large scale of resources
```

Comment:
can we make it clear based on the parameter of disableInformerCache

SOmething like if disableInformerCache = false => equivalent to existing velero behaviour
= true => better restore performance but higher memory usage.

Reply:
@anshulahuja98 How about this one?

Velero introduces the informer cache which is enabled by default. The informer cache improves the restore performance but may cause higher memory consumption. Increase the memory limit of the Velero pod or disable the informer cache by specifying the `--disable-informer-cache` option when installing Velero if you get the OOM error.

Reply:
Seems good to me. Thank you for taking this up.

Reply:
CC: @sseago as FYI